pymobiledevice3/usbmux.py

```diff
@@ -353,7 +353,14 @@ def get_device_list(self, timeout: float = None) -> None:
         """ get device list synchronously without waiting the timeout """
         self.devices = []
         self._send({'MessageType': 'ListDevices'})
-        for response in self._receive(self._tag - 1)['DeviceList']:
+        ret = self._receive(self._tag - 1)
```

Comment:
let's simply replace with `_send_receive()`. just add a `return response` to avoid code duplication